conbench/templates/benchmark-entity.html

```diff
@@ -30,6 +30,22 @@
 </div>
 <br>
 
+<div class="row">
+  <div class="col-md-12">
+    Click on a gray glyph in the plot to see corresponding benchmark run details.
+    <div class="conbench-histplot-run-details" style="display: none;">
+      <br />
+      <div class="panel panel-success">
+        <div class="panel-heading">Details for selected benchmark (green data point in plot)</div>
```

Comment:
I think this is good but I do think that we could standardize language - i.e. we use data point here and glyph elsewhere.

Reply:
thanks, updated to use "data point" instead of glyph